src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -59,7 +62,8 @@ public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolut
 
             this.errorListInfoBarController = this.extensionHost.GetService<IErrorListInfoBarController>();
             this.errorListInfoBarController.AssertLocalServiceIsNotNull();
-            this.errorListInfoBarController.Refresh();
+            
+            //TODO: check whether the errorListInfobarController needs to be refreshed
```

Comment:
Remove this comment as it was tested.

Reply:
Discussed offline: we tested that the error list controller didn't fail, but we haven't tested that the gold bar appears as expected.